Samples/2.3/HealthCheckSample/src/Grains/LocalHealthCheckGrain.cs

```diff
@@ -0,0 +1,12 @@
+using System.Threading.Tasks;
+using Orleans;
+using Orleans.Concurrency;
+
+namespace Grains
+{
+    [StatelessWorker]
```

Comment:
What do you think if this is marked so only 1 can spawn?

```suggestion
    [StatelessWorker(1)]
```